src/app/content/practiceQuestions/components/Answer/styled.tsx

```diff
@@ -9,18 +9,18 @@ export const AnswerExcerpt = styled.span`
   ${textRegularStyle}
   width: 100%;
   padding: 0;
-  overflow: auto;
-
-  * {
-    overflow: initial;
-  }
 `;
 
 // tslint:disable-next-line: variable-name
 export const AnswerAlignment = styled.div`
   display: flex;
   align-items: center;
   min-height: 4rem;
+  flex: 1;
+
+  * {
+    overflow: initial;
+  }
```

Comment:
I think you need `overflow: initial;` only in `AnswerContent`

Reply:
Without
```
* {
    overflow: initial;
  }
```
the math will not be displayed. It seems that MathJax doesn't like `overflow: auto`.

Reply:
I've meant leave this part where it was before and add `overflow: initial;` to `AnswerContent`